sdk/src/sysvars/fees.rs

```diff
@@ -96,5 +101,5 @@ impl Fees {
 }
 
 impl Sysvar for Fees {
-    impl_sysvar_get!(sol_get_fees_sysvar);
+    impl_sysvar_get!(FEES_ID, 7);
```

Comment:
The fees sysvar is not supported by `sol_get_sysvar`, so this should stay as it is: https://github.com/anza-xyz/agave/blob/8fe17e2a7cecbbc59a5954510d2abdf179fb6700/program-runtime/src/sysvar_cache.rs#L109